Libraries/Microsoft.Teams.Common/Http/HttpClient.cs

```diff
@@ -172,7 +173,7 @@ private async Task<object> ParseErrorBody(HttpResponseMessage response)
 
         try
         {
-            var bodyAsJson = JsonSerializer.Deserialize<Dictionary<string, object>>(content);
+            var bodyAsJson = JsonSerializer.Deserialize<Dictionary<string, JsonElement>>(content);
```

Comment:
this is unrelated to this PR, but It took me awhile to figure out why the error messages were not printed out, this happend when the json has a nested object, I believe that with JsonElement we won't loose the values..

And TBH, I would return just the string, since we cannot guarantee the server will always return errors as json.